cli_li3ds/import_orimatis.py

```diff
@@ -283,7 +283,7 @@ def datasource_image(session, referential, datasource,
     if image_file_ext:
         image_path = image_path.with_suffix(image_file_ext)
     image_path = os.path.normpath(str(image_path))
-    uri = 'file:{}'.format(image_path)
+    uri = 'file://{}'.format(image_path)
```

Comment:
I am fine with supporting `//`. It is just that `//` didn't look very good with `column`. Also, I am totally fine with using plpython and use urlparse for the check function.